app/models/spree/product_decorator.rb

```diff
@@ -128,6 +128,14 @@
 
   # -- Methods
 
+  def on_hand
+    if has_variants?
+      variants.map(&:on_hand).reduce(:+)
+    else
+      master.on_hand
+    end
+  end
```

Comment:
This mimics current Spree implementation. It was required for the specs not to fail. This `on_hand` attribute will be addressed in an upcoming PR.

Reply:
For my understanding, can you pls explain what do you mean by "will be addressed"?

Reply:
I didn't put to much thought on what the actual behaviour for Product#on_hand should be now that Spree removes it. Needs to be done in a different PR anyway.

It's likely that it'll delegate to master's on_hand.

Reply:
all right, thanks for explaining.